Clarify use with PING frames (#137)
quicwg/ack-frequency: draft-ietf-quic-ack-frequency.md

loss. The IMMEDIATE_ACK frame helps mitigate this problem.

An IMMEDIATE_ACK frame can be useful in other situations as well. For example,
if a sender wants an immediate RTT
measurement or if a sender wants to establish receiver liveness as quickly

> as possible. PING frames are ack-eliciting but if a PING frame is sent without

Author: ```suggestion
as possible. PING frames (Section 19.2 of {{QUIC-TRANSPORT}}) are ack-eliciting but if a PING frame is sent without
```

loss. The IMMEDIATE_ACK frame helps mitigate this problem.

An IMMEDIATE_ACK frame can be useful in other situations as well. For example,

> it can be send in the same packet together with a PING frame
> (Section 19.2 of {{QUIC-TRANSPORT}}) if a sender wants an immediate RTT

Reviewer: ```suggestion
if a sender wants an immediate RTT
```

> it can be send in the same packet together with a PING frame
> (Section 19.2 of {{QUIC-TRANSPORT}}) if a sender wants an immediate RTT
> measurement or if a sender wants to establish receiver liveness as quickly
> as possible. PING frames are ack-eliciting but if a PING frame is sent without
> an IMMEDIATE_ACK frame, the receiver might not immediately send an ACK based
> on it local ACK strategy.

Reviewer 2: ```suggestion
it can be sent as an alternative of a PING frame
(Section 19.2 of {{QUIC-TRANSPORT}}) if a sender wants an immediate RTT
measurement or if a sender wants to establish receiver liveness as quickly
as possible. PING frames are ack-eliciting but
the receiver might not immediately send an ACK based
on it local ACK strategy.
```

Maybe state that IMMEDIATE_ACK frames can be used as an alternative? I think that was the point of @neild.

> as possible. PING frames are ack-eliciting but if a PING frame is sent without
> an IMMEDIATE_ACK frame, the receiver might not immediately send an ACK based
> on it local ACK strategy.

Author: ```suggestion
as possible. PING frames (Section 19.2 of {{QUIC-TRANSPORT}}) are ack-eliciting
but if a PING frame is sent, the receiver might not immediately send an ACK based
on it local ACK strategy.
```
